classes/02class/exercises/c02-network03/rodrigovcesar/README.md

```diff
@@ -0,0 +1,73 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```
+aws ec2 create-vpc --cidr-block 10.0.0.0/16 --instance-tenancy default 
```

Comment:
No need to specify "tenancy" default, but no harm in doing it.

Also, nothing wrong in using `10.0.0.0/16` but in my personal experience this is so easy to clash with other networks in the future (if you ever need to connect them).